README.md

```diff
@@ -51,7 +51,7 @@ It expects one argument to be passed as a param.
 ```
 
 ```handlebars
-<div class="{{if (is-mobile 'android' 'is-android')}} {{if (is-mobile 'apple' 'is-apple')}}">
+<div class="{{if (is-mobile 'android') 'is-android')}} {{if (is-mobile 'apple') 'is-apple')}}">
```

Comment:
This still doesn't work. You still have the extra parentheses on the ends there.
Should be: 
```hbs
<div class="{{if (is-mobile 'android') 'is-android'}} {{if (is-mobile 'apple') 'is-apple'}}">
```

Reply:
My bad, sorry for multiple commits